src/modules/api/controllers/debug.controller.ts

```diff
@@ -70,12 +70,29 @@ export class DebugApiController {
   @Get('/peerlist')
   async getPeerList() {
     try {
-      const peers = await ipfsContainer.ipfs.swarm.peers()
+      const peers = await ipfsContainer.self.swarm.peers()
       const peerAddresses = peers.map((item) => item.peer)
 
       return peerAddresses
     } catch (err) {
       throw new InternalServerErrorException(`Error getting peer list ${err.message}`)
     }
   }
+
+  @Get('/dumpdatabase')
+  async dumpDatabase() {
```

Comment:
Only thing to note here is we might want to include `graph.cs` at some point. For now this is definitely enough.